social-ui/src/ForYouCard.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import Meta from "./Meta";
+import yourArticles from "./_data/your-articles.json";
+
+class CardFor extends Component {
+  constructor() {
+    super();
+    this.state = {
+      yourArticles
+    };
+  }
+
+  render() {
+    return this.state.yourArticles.map((article, idx) => (
```

Comment:
Good use of map, but remember that anywhere it is used for react rendering is a good indication that you should use a reusable component.